android/src/main/java/com/thebluealliance/androidclient/binders/RecyclerViewBinder.java

```diff
@@ -0,0 +1,127 @@
+package com.thebluealliance.androidclient.binders;
+
+import android.support.annotation.Nullable;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.View;
+import android.widget.ProgressBar;
+
+import com.thebluealliance.androidclient.Constants;
+import com.thebluealliance.androidclient.R;
+import com.thebluealliance.androidclient.adapters.ListViewAdapter;
+import com.thebluealliance.androidclient.listitems.ListItem;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import io.nlopez.smartadapters.SmartAdapter;
+import io.nlopez.smartadapters.adapters.RecyclerMultiAdapter;
+
+public class RecyclerViewBinder extends AbstractDataBinder<List<Object>> {
+
+    @Bind(R.id.list)
+    RecyclerView mRecyclerView;
+    @Bind(R.id.progress)
+    ProgressBar mProgressBar;
+
+    private RecyclerViewBinderMapper mMapper;
+
+    private RecyclerMultiAdapter mAdapter;
+
+    public void setRecyclerViewBinderMapper(RecyclerViewBinderMapper mapper) {
+        mMapper = mapper;
+    }
+
+    @Override
+    public void bindViews() {
+        ButterKnife.bind(this, mRootView);
+    }
+
+    @Override
+    public void updateData(@Nullable List<Object> data) {
+        if (data == null || mRecyclerView == null) {
+            setDataBound(false);
+            return;
+        }
+        if (data.isEmpty()) {
+            setDataBound(false);
+            return;
+        }
+        if (mMapper == null) {
+            throw new RuntimeException(this.getClass().getName() + " does not have a RecyclerViewBinderMapper set!");
```

Comment:
Can we handle this more gracefully? Throwing a `RuntimeException` will cause crashes. Can we just complain via `Log.e` and bind the noDataView instead?